app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "test@example.com"
```

Comment:
it would be good to have this email in config. I see we don't have a separate config, but we use `secrets.yml` for that, so you can add it there.